components/Voting/sessionPanel.styled.tsx

```diff
@@ -0,0 +1,84 @@
+/* tslint:disable:object-literal-sort-keys */
+/* tslint:disable:object-literal-key-quotes */
+import { breakpoint } from '../utils/styles/breakpoints'
+import { calcRem } from '../utils/styles/calcRem'
+import styled from '../utils/styles/theme'
+
+export const Panel = styled('li')(({ theme }) => ({
+  backgroundColor: theme.colors.lightGrey,
+  marginTop: '10px',
+  padding: '13px 15px',
+  listStyle: 'none',
+
+  '&:before': {
+    content: 'normal',
```

Comment:
Really need to get rid of that default style on lists...

Reply:
Is this a change you want me to make, or just a comment?

Reply:
That was just thinking out loud. It would be nice to do so we didn't have to do the content part and make life easier